packages/@aws-cdk/aws-neptune/lib/cluster.ts

```diff
@@ -362,7 +366,7 @@ export abstract class DatabaseClusterBase extends Resource implements IDatabaseC
 
   public grantConnect(grantee: iam.IGrantable): iam.Grant {
     if (this.enableIamAuthentication === false) {
-      throw new Error('Cannot grant connect when IAM authentication is disabled');
+      throw new Error('Cannot grant when IAM authentication is disabled');
```

Comment:
This error message doesn't quite make sense worded this way.

Reply:
Actually, I think this wording change should go in the other PR anyway.

Reply:
Dropped from here, and will address separately in the other PR